lib/parser.js

```diff
@@ -16,6 +17,9 @@ const PARSERS = {};
 const xParserCircle = 'x-parser-circular';
 const xParserMessageParsed = 'x-parser-message-parsed';
 
+const schemaValidationErrors = [];
```

Comment:
My 50 cents, where do you "clear" that errors? As I see in the changed code, that array preserve state between `parse` execution.